frontend/src/bundles/home/components/filter/components/dropdown-item/dropdown-item.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback } from 'react';
+
+import { BaseButton } from '~/bundles/common/components/components';
+
+import styles from './styles.module.scss';
+
+type DropdownItemProperties = {
+    optionId: string;
+    label: string;
+    onOptionSelect: (optionId: string) => void;
+};
+
+const DropdownItem: React.FC<DropdownItemProperties> = ({
+    optionId,
+    label,
+    onOptionSelect,
+}) => {
+    const handleSelectOption = useCallback(() => {
+        onOptionSelect(optionId);
+    }, [optionId, onOptionSelect]);
+
+    return (
+        <li className={styles.dropdown_content__item}>
+            <BaseButton onClick={handleSelectOption}>{label}</BaseButton>
```

Comment:
Why do you need a button here? Check if dropdown have props like onChange?

Reply:
i don't use dropdown because i get this. I can't change styles because it default from react-select
![select](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/120641395/93d29a60-e7a7-4f9a-a84d-110c4f639452)

Reply:
Actually you can. As far as I know this package allows you to customize all dropdown elements